### PR TITLE
[KYLIN-4627] Add the document about try kylin with docker to help users to try kylin easily

### PR DESCRIPTION
**Pull request by zhouyao1994**
## Proposed changes

[KYLIN-4627] Add the document about try kylin with docker to help users to try kylin easily

## Types of changes

What types of changes does your code introduce to Kylin?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Documentation Update (if none of the other choices apply)

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._

- [x ] I have create an issue on [Kylin's jira](https://issues.apache.org/jira/browse/KYLIN), and have described the bug/feature there in detail
- [x] Commit messages in my PR start with the related jira ID, like "KYLIN-0000 Make Kylin project open-source"
- [x] Compiling and unit tests pass locally with my changes
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] If this change need a document change, I will prepare another pr against the `document` branch
- [x] Any dependent changes have been merged

## Further comments

If this is a relatively large or complex change, kick off the discussion at user@kylin or dev@kylin by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by RupengWang** [comment]
+1

File: README.md [hunk 1]
```diff
@@ -14,11 +14,37 @@ Apache Kylin
...
 Apache Kylin is an open source Distributed Analytics Engine, contributed by eBay Inc., provides SQL interface and multi-dimensional analysis (OLAP) on Hadoop supporting extremely large datasets.
 
-For more details, see the website [http://kylin.apache.org](http://kylin.apache.org).
+For more details, see the website [http://kylin.apache.org](http://kylin.apache.org), Chinese version:[http://kylin.apache.org/cn/](http://kylin.apache.org/cn/).
+
+Get start kylin in 5 min with Docker
+=============
+In order to allow users to  try Kylin easily, We provide Kylin’s docker image.
+
+Just run codes blow in your terminal. After 3~5 mins, you can access kylin WebUI [http://127.0.0.1:7070/kylin/login](http://127.0.0.1:7070/kylin/login) in your browser.
```
Comment (hit-lacus) [inline README.md:23]: Typo, should be "below" .

File: README.md [hunk 2]
```diff
@@ -14,11 +14,37 @@ Apache Kylin
 
 Apache Kylin is an open source Distributed Analytics Engine, contributed by eBay Inc., provides SQL interface and multi-dimensional analysis (OLAP) on Hadoop supporting extremely large datasets.
 
-For more details, see the website [http://kylin.apache.org](http://kylin.apache.org).
+For more details, see the website [http://kylin.apache.org](http://kylin.apache.org), Chinese version:[http://kylin.apache.org/cn/](http://kylin.apache.org/cn/).
+
+Get start kylin in 5 min with Docker
+=============
+In order to allow users to  try Kylin easily, We provide Kylin’s docker image.
```
Comment (hit-lacus) [inline README.md:21]: lowercase the word "We"

File: README.md (hunk 1 above)
Comment (zhouyao1994) [reply README.md]: thx

File: README.md (hunk 2 above)
Comment (zhouyao1994) [reply README.md]: thx

**Comment by hit-lacus** [comment]
LGTM

**Pull request merged**